echopype/echodata/sensor_ep_version_mapping/v05x_to_v06x.py

```diff
@@ -0,0 +1,1091 @@
+            ed_obj[grp_path] = ed_obj[grp_path].rename({"frequency": "channel"})
+
+            # set values for channel
+            if "channel_id" in ed_obj[grp_path]:
+                ed_obj[grp_path]["channel"] = ed_obj[grp_path].channel_id.values
+                ed_obj[grp_path] = ed_obj._tree[grp_path].ds.drop("channel_id")
+
+            else:
+                ed_obj[grp_path]["channel"] = channel_id.sel(
+                    frequency=ed_obj[grp_path].frequency_nominal
+                ).values
+
+            # set attributes for channel
+            ed_obj[grp_path]["channel"] = ed_obj[grp_path]["channel"].assign_attrs(
+                _varattrs["beam_coord_default"]["channel"]
+            )
+
+
+def _change_beam_var_names(ed_obj, sensor):
+    """
+    For EK60 ``Beam_group1``
+    1. Rename ``beamwidth_receive_alongship`` to
+    ``beamwidth_twoway_alongship`` and change the attribute
+    ``long_name``
+    2. Rename ``beamwidth_transmit_athwartship`` to
+    ``beamwidth_twoway_athwartship`` and change the attribute
+    ``long_name``
+    3. Remove the variables ``beamwidth_receive_athwartship``
+    and ``beamwidth_transmit_alongship``
+    4. Change the attribute ``long_name`` in the variables
+    ``angle_offsetalongship/athwartship`` and
+    ``angle_sensitivity_alongship/athwartship``
+
+    For EK80 ``Beam_group1``
+    1. Change the attribute ``long_name`` in the variables
+    ``angle_offset_alongship/athwartship``
+
+    Parameters
+    ----------
+    ed_obj : EchoData
+        EchoData object that was created using echopype version 0.5.x.
+    sensor : str
+        Variable specifying the sensor that created the file.
+
+    Notes
+    -----
+    The function directly modifies the input EchoData object.
+    """
+
+    if sensor == "EK60":
+
+        ed_obj["Sonar/Beam_group1"] = ed_obj["Sonar/Beam_group1"].rename(
+            {"beamwidth_receive_alongship": "beamwidth_twoway_alongship"}
+        )
+
+        ed_obj["Sonar/Beam_group1"].beamwidth_twoway_alongship.attrs[
+            "long_name"
+        ] = "Half power two-way beam width along alongship axis of beam"
+
+        ed_obj["Sonar/Beam_group1"] = ed_obj["Sonar/Beam_group1"].rename(
+            {"beamwidth_transmit_athwartship": "beamwidth_twoway_athwartship"}
+        )
+
+        ed_obj["Sonar/Beam_group1"].beamwidth_twoway_athwartship.attrs[
+            "long_name"
+        ] = "Half power two-way beam width along athwartship axis of beam"
+
+        ed_obj["Sonar/Beam_group1"] = ed_obj["Sonar/Beam_group1"].drop(
+            ["beamwidth_receive_athwartship", "beamwidth_transmit_alongship"]
+        )
+
+        ed_obj["Sonar/Beam_group1"].angle_sensitivity_alongship.attrs[
+            "long_name"
+        ] = "alongship angle sensitivity of the transducer"
+
+        ed_obj["Sonar/Beam_group1"].angle_sensitivity_athwartship.attrs[
+            "long_name"
+        ] = "athwartship angle sensitivity of the transducer"
```

Comment:
Small things, but for EK80 in v0.5.x both of these miss one word "angle" in it, so it seems a good idea to just reset:
https://github.com/OSOceanAcoustics/echopype/blob/bc8afa190fa2ca4fab5944bac83cd4b20f7abcf6/echopype/convert/set_groups_ek80.py#L300-L309

Reply:
So, you are suggesting to do a PR that removes "angle" from EK60. Or we could just add "angle" to EK80.

Reply:
I am thinking to add "angle" to EK80.

Reply:
Ok, I will go ahead and do this.